dist/components/data/info_window.js

```diff
@@ -1,13 +1,14 @@
 'use strict';
-define(['jquery', 'flight/lib/component', 'map/components/mixins/map_utils'], function($, defineComponent, mapUtils) {
+define(['jquery', 'flight/lib/component'], function($, defineComponent) {
   var infoWindowData;
   infoWindowData = function() {
     this.defaultAttrs({
-      pinRoute: "/map/pin/"
+      route: "/map/pin/",
+      refinements: ''
```

Comment:
A query string makes it difficult for the app to implement. It's only used to make the post request, which is an internal part of the info window component.